src/modules/map/components/openlayers/utils/useMapInteractions.composable.js

```diff
@@ -20,6 +20,8 @@ import log from '@/utils/logging'
 const dispatcher = {
     dispatcher: 'useMapInteractions.composable',
 }
+// these are the events that can trigger a long press event. This is necessary for IoS support
+const longPressEvents = ['touch', '', 'pen']
```

Comment:
Maybe split your comment in multiple lines, something like
```Javascript
// these are the events that can trigger a long press event.
const longPressEvents = [
    'touch',
     'pen',
    '', // This is necessary for iOS support
]
```
As it's not clear currently if all are necessary for iOS support, or which one of them is

Reply:
yeah, the comment predates the time I had a good name for the variable ^^.